adafruit_led_animation/group.py

```diff
@@ -152,10 +152,18 @@ def animate(self, show=True):
                         member.show()
             return result
 
+        # improve performance of non-synced animations by only calling show on the last member
         ret = False
+        pos = 1
+        num_members = len(self._members)
         for item in self._members:
-            if item.animate(show):
-                ret = True
+            if pos == num_members:
+                if item.animate(show):
+                    ret = True
+            else:
+                if item.animate(False):
+                    ret = True
+                pos += 1
```

Comment:
It's somewhat minor, but I think this code is a litter more forgiving if it would use `enumerate()` with the for loop instead of incrementing the `pos` variable manually. If someone were to accidentally remove the manual incrementing it would cause the condition to never be true, with enumerate it "attaches" it to the for loop mechanism so it doesn't require any separate step in the code.

```
for index, item in enumerate(self._members):
    if index == num_members-1:
        if item.animate(show):
            ret = True
    else:
        if item.animate(False):
            ret = True
```

Reply:
I would agree. I've updated to the more robust implementation